modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/VerticalNav.js

```diff
@@ -43,6 +43,10 @@ export default function VerticalNav({
 		>
 			{(item) => (
 				<ClayVerticalNav.Item
+
+					// To remove active={active === item.id} when it is fixed on Clay https://github.com/liferay/clay/issues/5595
+
+					active={active === item.id}
```

Comment:
Do we need the flag here in the item in addition to passing it as key in the parent?